perun-engine/src/main/java/cz/metacentrum/perun/engine/processing/impl/EventParserImpl.java

```diff
@@ -36,7 +36,7 @@ public Task parseEvent(String event) throws InvalidEventMessageException,
 			ServiceNotExistsException, InternalErrorException,
 			PrivilegeException {
 
-		log.info("Going to process event: {}", event);
+		log.debug("Going to process event: {}", event);
```

Comment:
I would keep this also at info, since the rest of messages about receiving and parsing the message are at debug level. So if debug is disable, we can't tell if some message was received and what was the content.